### DISCUSSION TITLE
Change wireless LAN/SSID

### DISCUSSION THREAD

**Discussion by Fish31520** [mcguirepr89/BirdNET-Pi#733] [Q&A]
Hi ! 2nd question for today 😅
Is there a way to change the wifi spot used to ssh Birdnet-Pi (I'm about to change my internet provider) ?
Thanks !

**Comment by JohnJoosten** [comment]
From Birdnet interface use Tools, Webterminal and login to your webterminal on your PI 
type "sudo Raspi-Config"
select 1 System Options
select S1 Wireless LAN
enter SSID and password
use "finish"  on main menu to save changes;

More info can be found via https://www.seeedstudio.com/blog/2021/01/25/three-methods-to-configure-raspberry-pi-wifi/

**Reply by Fish31520** [reply]
@JohnJoosten
Thanks a lot !

**Reply by tom-heisenlin** [reply]
Hi,
unfortunately the hint from JohnJoosten is not working for me:

I have to use "sudo raspi-config" to enter into the config.
following the mentioned steps..... using "finish" .... then reboot, brings me back into the initial setup - no changes !
so, what am I doing wrong ?

I do not only want to change the SSID, but also want to give a fixed IP to the Raspi - how to do it ?

thanks for you support

Tom
